### DISCUSSION TITLE
2022適合學的程式語言

### DISCUSSION THREAD

**Discussion by Pietra1226** [1101-GS4719-C/Python-Community-GS4719-C#785] [General]
雖然這個主題很多人做過，而且老實說適合學什麼程式語言都是因人而異，
但這部影片確實更讓我明確的搞清楚自己想要的是什麼。
這部是藉由未來工作接觸的方向來提供你應該學或是適合學的程式語言。
# 懶人包
1. 去年最熱門的前三是 JavaScript、Python、Java。
2. 對Web跟瀏覽器有興趣可以學JavaScript。同時可以考慮學TypeScript。
3. 對後端、ML、data、DL和AI有興趣可以學python。
4. 對手機App有興趣可以學Swift、Java
5. 對加密貨幣或比較base的可以考慮C++

上面是我大概對影片的摘要
影片連結: [https://www.youtube.com/watch?v=pSoDUFdqVOU&t=508s](https://www.youtube.com/watch?v=pSoDUFdqVOU&t=508s)

**Comment by ruthchen1020** [comment]
學工程都學C的系列，但python真的是大勢。

**Comment by StevenWanglolz** [comment]
感謝統整! 這樣對未來有個大概的方向了!

**Comment by swerfg03joe** [comment]
這通常都是寫給程式小白看的，不同的工作當然要選擇不同的工具，程式語言常會被認為是高不可攀的學科，但真正接觸後你會發現任何語言操作方法大同小異，差別只在於你需要做什麼工作,例如python已經將整個Machine Learning的套件打包好，我們只需要像疊積木一樣將所需的功能插入，而不是用C語言從頭開始做模型
